standard/lpdb.lua

```diff
@@ -258,4 +258,30 @@ Lpdb.Placement = Model('placement', {
 	{name = 'extradata', fieldType = 'struct', default = {}},
 })
 
+---@class SquadPlayerModel:Model
+Lpdb.SquadPlayer = Model('squadplayer', {
+	{
+		name = 'objectname',
+		fieldType = 'string',
+		default = function(fields)
+			return fields.link .. '_' .. (fields.joindate or '') .. '_' .. (fields.role or '') .. '_' .. (fields.status or '')
```

Comment:
Is the sanitizing that was present before (ReferenceCleaner, resolve_redirect, link or id) done earlier (with the refactoring)?

Reply:
Link should be set with resolve redirect
https://github.com/Liquipedia/Lua-Modules/blob/e4825518bd17d46e6d2eb122ba54a0f19fc56777/components/squad/commons/squad_row.lua#L103

Dates are cleaned with Reference Cleaner
https://github.com/Liquipedia/Lua-Modules/blob/e4825518bd17d46e6d2eb122ba54a0f19fc56777/components/squad/commons/squad_row.lua#L194

Reply:
Looks good, thx!
Is the removal of `mw.title.getCurrentTitle().prefixedText` intended?

Reply:
Yeah, it serves no real purpose, as it has no impact of the uniqueness of the `objectname` (as all objectnames are automatically prefered with `pageid`)